### ISSUE TITLE
Syntax highlighting not working in jupyter lab v3.0.16

### ISSUE THREAD

**Issue by jtackm** [JuliaLang/IJulia.jl#1013]
After upgrading jupyter + jupyter lab, IJulia no longer shows correct syntax highlighting within notebooks (examples: missing string highlighting, functions and arguments have the same color, closing brackets sometimes have different colors than opening brackets, some closing brackets highlighted as in red when under cursor even though they are correct). 

Jupyter setup:
```
jupyter core     : 4.7.1
jupyter-notebook : 6.4.0
qtconsole        : not installed
ipython          : 7.25.0
ipykernel        : 5.5.5
jupyter client   : 6.1.12
jupyter lab      : 3.0.16
nbconvert        : 6.1.0
ipywidgets       : not installed
nbformat         : 5.1.3
traitlets        : 5.0.5
```

Julia setup:
```
(@v1.6) pkg> st IJulia
      Status `~/.julia/environments/v1.6/Project.toml`
  [7073ff75] IJulia v1.23.2

julia> versioninfo()
Julia Version 1.6.0
Commit f9720dc2eb (2021-03-24 12:55 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: AMD EPYC 7742 64-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, znver2)
```

Downgrading jupyterlab to v3.0.0 solves this issue (haven't tested intermediate versions).

**Comment by baggepinnen** [comment]
This just happened to me, and this comment solved it for me
https://stackoverflow.com/a/69545072/1914791

TLDR: just refresh the browser tab :)

**Comment by jtackm** [comment]
Thanks! I'm currently running a setup similar to @urlicht and have not been having issues for a while.
